docs/articles/documentation/test-api/selecting-page-elements/selectors.md

```diff
@@ -620,16 +613,56 @@ DOM node or the timeout exceeds.
 Note that you can additionally require that the node returned by the selector is visible.
 To do this, use the [visibilityCheck](selector-options.md#optionsvisibilitycheck) option.
 
+## Adding Custom Properties to Element State
+
+TestCafe allows you to extend [element state](#obtain-element-state) with custom properties calculated on the client side.
+
+To do this, use the selector's `addCustomDOMProperties` method.
+
+```text
```

Comment:
In these cases we write a formal description, so it should be something like:
`Selector().addCustomDOMProperties( { [...propertyHandlers] } );` or
`Selector().addCustomDOMProperties( { property1 => fn1, ... propertyN => fnN } );`
Besides, several strings of code without highlighting look awful in my opinion.